c/Cargo.toml

```diff
@@ -18,6 +18,7 @@ env_logger = "0.11.5"
 foxglove = { path = "../rust/foxglove", default-features = false, features = [
   "live_visualization",
   "lz4",
+  "tls",
```

Comment:
@ericmlujan the bridge uses the pre-built libraries rather than building from source itself, right? So we need to include this here?

Reply:
Bridge uses pre-built binaries by default, so I'd include this (there is a mode for also using a locally built Rust SDK, which we use in CI and while developing, but our users are highly unlikely to use that)